Package.swift

```diff
@@ -44,7 +44,8 @@ var targets: [PackageDescription.Target] = [
             .product(name: "ServiceDiscovery", package: "swift-service-discovery"),
             .product(name: "Backtrace", package: "swift-backtrace"),
             .product(name: "AsyncAlgorithms", package: "swift-async-algorithms"),
-        ]
+        ],
+        swiftSettings: [.swiftLanguageMode(.v5)]
```

Comment:
Decided to put it per target for future migration to strict concurrency

Reply:
Agreed, we’ll go one by one here